### DISCUSSION TITLE
Save the running server version

### DISCUSSION THREAD

**Discussion by issue42** [thijsvanloef/palworld-server-docker#289] [General]
to configure whether to automatically update the server version instead of checking and updating it every time you start it.

**Comment by teeeebs** [comment]
Could you please rephrase the question?

**Reply by issue42** [reply]
Sorry now i see the config.
I thought the configuration didn't take effect.
Its my config erro.
Maybe I didn't configure the variables.

**Reply by teeeebs** [reply]
no worries! Just wanted to make sure that you figured it out?
